docs/advanced-topics/reproducing.md

```diff
@@ -53,6 +53,13 @@ OSS-Fuzz, then feed the reproducer input to the fuzz target ([how?]({{
 site.baseurl }}/getting-started/new-project-guide/#prerequisites), [why?]({{
 site.baseurl }}/faq/#why-do-you-use-docker)).
 
+Reproducing with docker is very likely to "just work". But if you are a project
+maintainer who is unfamiliar with OSS-Fuzz and docker trying to repro a crash in
+a hurry, you may want to try building your project with clang and the relevant
+sanitizer and fuzzer using your project's standard build process. If that fails,
```

Comment:
This seems a bit repetitive given the previous paragraph, which already mentions building using the project's build system. Can we make that paragraph a bit more prominent instead of adding this one?